synapseclient/client.py

```diff
@@ -1156,21 +1262,25 @@ def move(self, entity, new_parent):
 
         return entity
 
-    def _getWithEntityBundle(self, entityBundle, entity=None, **kwargs):
+    def _getWithEntityBundle(
+        self, entityBundle: dict, entity: Entity = None, **kwargs
+    ) -> Entity:
         """
-        Creates a :py:mod:`synapseclient.Entity` from an entity bundle returned by Synapse.
+        Creates a [Entity][synapseclient.Entity] from an entity bundle returned by Synapse.
         An existing Entity can be supplied in case we want to refresh a stale Entity.
 
-        :param entityBundle: Uses the given dictionary as the meta information of the Entity to get
-        :param entity:       Optional, entity whose local state will be copied into the returned entity
-        :param submission:   Optional, access associated files through a submission rather than
-                             through an entity.
+        Arguments:
+            entityBundle: Uses the given dictionary as the meta information of the Entity to get
+            entity:       Optional, entity whose local state will be copied into the returned entity
+            submission:   Optional, access associated files through a submission rather than through an entity.
 
-        See :py:func:`synapseclient.Synapse.get`.
-        See :py:func:`synapseclient.Synapse._getEntityBundle`.
-        See :py:mod:`synapseclient.Entity`.
-        """
+        Returns:
+            A new Synapse Entity
 
+        See [get][synapseclient.Synapse.get].
```

Comment:
The best way to have a list like this is to use `-` to make them an unordered list, like:

```
- See ...
- See ....
- See ...
```